sql/information_schema/information_schema.go

```diff
@@ -1993,6 +1991,30 @@ func partitionKey(tableName string) []byte {
 	return []byte(InformationSchemaDatabaseName + "." + tableName)
 }
 
+func getColumnDefaultString(cd *ColumnDefaultValue) interface{} {
```

Comment:
1) Add a method signature
2) It's not always returning a string so getColumnDefaultValue is probably better